### PR TITLE
Have the ability to set a Read-Only user for queries

### PR DESCRIPTION
**Pull request by glasnt**
Use the RO credentials for all queries, and the RW for importing and categorization updates.

### CODE REVIEW

**Comment by Hobbsee** [comment]
Question - should RO or RW be the default connection here? i.e. should we have an @influx and an @influx_rw rather than an @influx_ro and an @influx?  Given so much of this is about presenting data rather than changing it, I'd lean towards the former (and as a bonus, it's safer!).

Looks good otherwise!

**Comment by glasnt** [comment]
Ideally we'd like to have one default user for this app, so the minimum permissions for a single-user instance would be R/W. Hence, R/O being optional and named as such. 

I will create some basic doco to create both users, and update the README

**Comment by nerdyness** [comment]
> should we have an @influx and an @influx_rw rather than an @influx_ro and an @influx?

Agree, it should be `influx` and `influx_rw`. The app needs one main connection to the influxdb (`influx`) and on the edge case that somebody writes to the categories, a new connection with `influx_rw` can be opened.

**Comment by glasnt** [comment]
Logic, configs, and readme updated to reflect the polarity reversal.

File: README.md [hunk 1]
```diff
@@ -13,6 +13,29 @@ Pigeonhole requires the following:
 
 After these have been installed, copy the config.toml.example file to config.toml, and update it with your details.
 
+Pigeonhole operates as a ro user with the exception of the categorisation. If you're using the categorisation feature you'll have to add a rw user as well.
```
Comment (nerdyness) [inline README.md]: Given that this is public, I'd:
`s/ro user/read-only user/ s/rw user/read-write user/`

**Comment by nerdyness** [comment]
Looks good albeit the doco I mentioned :+1:

**Pull request merged**